web/cobrands/fixmystreet/staff.js

```diff
@@ -299,6 +299,21 @@ $.extend(fixmystreet.set_up, {
         toggle_public_update();
     });
 
+    $('#detailed_information').on('keyup', function() {
+        var $this = $(this),
+        counter = $('#detailed_information_length');
+        if (!$this.data('max-length')) {
```

Comment:
This could be done outside the function, so the keyup is only set up if the data is present.